src/playback/view/mixerpanelmodel.cpp

```diff
@@ -40,22 +40,24 @@ void MixerPanelModel::load()
 {
     TrackSequenceId sequenceId = controller()->currentTrackSequenceId();
 
-    playback()->tracks()->trackIdList(sequenceId)
-    .onResolve(this, [this, sequenceId](const TrackIdList& trackIdList) {
-        loadItems(sequenceId, trackIdList);
-    })
-    .onReject(this, [sequenceId](int errCode, std::string text) {
-        LOGE() << "unable to find track sequence:" << sequenceId << ", error code: " << errCode
-               << ", " << text;
-    });
-
-    playback()->tracks()->trackAdded().onReceive(this, [this](const TrackSequenceId sequenceId, const TrackId trackId) {
-        addItem(sequenceId, trackId);
-    });
-
-    playback()->tracks()->trackRemoved().onReceive(this, [this](const TrackSequenceId /*sequenceId*/, const TrackId trackId) {
-        removeItem(trackId);
-    });
+    if (playback()->tracks()) {
```

Comment:
just put

if (playback()->tracks()) {
    return;
}

at the beginning

Reply:
I've pushed your suggestion.